app/controllers/api/v1/users_controller.rb

```diff
@@ -1,9 +1,26 @@
 class Api::V1::UsersController < ApiController
 
+  swagger_controller :users, 'Users'
+
+  swagger_api :index do
+    summary 'Responds with a list of users'
+    notes 'This lists all users in the system'
+    response :unauthorized
+  end
+
   def index
     @users = User.all
   end
 
+  swagger_api :show do
+    summary 'Responds with a user resource'
+    param :path, :user, :string, :required, "User Id"
```

Comment:
Are we requiring the resource is as `"1"` or as `1` ? If the latter them we should probably change `:string` to `:integer` or `: fixnum`